README.md

```diff
@@ -1,2 +1,161 @@
-# tajemouti.github.io
-Portfolio
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+**[Portfolio]** is a learning project in which it is required to implement HTML/CSS linters.. And to build a professional portfolio webpage based on Figma template..
+Click [here](https://tajemouti.github.io) to access the portfolio homepage.
+## 🛠 Built With <a name="built-with"></a>
+Visual Studia Code, Figma and HTML/CSS linters 
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Linters implementation]**
+- **[Portfolio homepage]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://drive.google.com/file/d/1CEg59wZK7R-0b2Kgi4WgXtW2oM2zAlDi/view?usp=sharing)
```

Comment:
- [ ] [OPTIONAL] For easy navigation I'd suggest you place the link to the live demo in the "Live Demo" section instead. For the video demonstration, you can create a sub-section under the "Live Demo" section and place it there as it might confuse users. 😉